### DISCUSSION TITLE
Wazuh Gmail Alerts

### DISCUSSION THREAD

**Discussion by BP-DSM** [Security-Onion-Solutions/securityonion#6343] [Unsupported Versions] [answered]
Hi all,

I am trying to configure my Security Onion Wazuh to send me email alerts when there are incidents, however no email was received when I mimick brute force attacks. I am using gmail account at the end point. I am trying to use gmail to send alerts without any authentication. https://support.google.com/a/answer/176600?hl=en#zippy=%2Cuse-the-gmail-smtp-server%2Cuse-the-restricted-gmail-smtp-server
P.S I have tried both smtp.gmail.com and aspmx.l.google.com but still no email was sent to me. Any help is appreciated.

Configuration snippet from /opt/so/conf/wazuh/ossec.conf
<global>
      3     <jsonout_output>yes</jsonout_output>
      4     <alerts_log>yes</alerts_log>
      5     <logall>yes</logall>
      6     <logall_json>yes</logall_json>
      7     <email_notification>yes</email_notification>
      8     <email_to>my@gmail.com</email_to>
      9     <smtp_server>smtp.gmail.com</smtp_server>
     10     <email_from>onion@onion.com</email_from>
     11     <email_maxperhour>50</email_maxperhour>
     12     <do_not_delay />
     13   </global>
     14   <alerts>
     15     <log_alert_level>1</log_alert_level>
     16     <email_alert_level>5</email_alert_level>
     17   </alerts>

**Comment by dougburks** [comment, marked as answer]
Check your Wazuh log for additional clues, but this seems to be an issue with Gmail configuration.

**Reply by BP-DSM** [reply]
Hi Dougburks,

I have tried to grep any error logs from Wazuh logs but no errors was found. Is there any Gmail configuration that is required?
Because I need inform my domain mail admin about it.

**Reply by dougburks** [reply]
Gmail configuration is beyond the scope of this discussion forum. Make sure that you're checking the Wazuh server logs at /nsm/wazuh/logs/.

**Comment by BP-DSM** [comment]
Thanks dougburks. Problem has been resolved.
